scrapers/xbvrdb.py

```diff
@@ -48,26 +65,76 @@ def find_scene_id(title):
             if 'originals' in t:
                 t.remove('originals')
             title='%'.join(t)+'%'
-        c.execute('select id from scenes where title like ?',(title+'%',))
+        c.execute('SELECT id FROM scenes WHERE title LIKE ?',(title+'%',))
         id=c.fetchone()
         if id is not None:
             return id[0]
     else:
         return id[0]
     return None
 
-if not path.exists("xbvr.db"):
+def query_api(filename):
+    request_config={"dlState":"available","cardSize":"1","lists":[],"isAvailable":True,"isAccessible":True,"isHidden":False,"isWatched":None,"releaseMonth":"","cast":[],"sites":[],"tags":[],"cuepoint":[],"attributes":[],"volume":0,"sort":"release_desc","offset":0,"limit":1}
+    response = requests.post(XBVR_HOST+'/api/scene/list', json=request_config)
```

Comment:
`requests` throws a lot of exceptions on dns, timeout, general connection errors. This shouldnt happen on same lan based hosts but doesnt hurt to avoid that with a try? Same for L85